lib/index.js

```diff
@@ -89,7 +89,8 @@ function createServer(options) {
             e
         ) {
             if (
-                this.listeners('uncaughtException').length > 1 ||
+                (opts.handleUncaughtExceptions &&
```

Comment:
Isn't this already gated (`opts.handleUncaughtExceptions`) on L84?

Reply:
Yeah, it is indeed. I'll remove this check.